constructorio_python/constructorio.py

```diff
@@ -40,7 +40,8 @@ def __init__(self, options) -> None:
             'service_url': service_url or 'https://ac.cnstrc.com',
             'requests': requests,
         }
-        self.autocomplete = Autocomplete()
+
+        self.autocomplete = Autocomplete(self.__options)
         self.search = Search()
         self.browse = Browse()
         self.recommendations = Recommendations()
```

Comment:
Should we be exposing the submodules at this point, since they aren't actually available yet?